src/Multisig.sol

```diff
@@ -415,5 +419,6 @@ contract Multisig is IMultisig, IMembership, PluginUUPSUpgradeable, ProposalUpgr
     /// @dev This empty reserved space is put in place to allow future versions to add new
     /// variables without shifting down storage in the inheritance chain.
     /// https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+    /// Q(danielw): will `multisigSettings` and `lastMultisigSettingsChange` share the same slot?
```

Comment:
As far as I can see, there should be 3 storage slots, each being:
- `mapping(uint256 => Proposal) internal proposals`
- `MultisigSettings public multisigSettings` where `bool + uint16 and uint64` should only consume one slot
- `uint64 public lastMultisigSettingsChange`

Did I miss something?